tf-modules/archive/async_operation.tf

```diff
@@ -17,9 +17,13 @@ resource "aws_ecs_task_definition" "async_operation" {
       {
         "name": "AWS_REGION",
         "value": "${data.aws_region.current.name}"
+      },
+      {
+        "name": "databaseCredentialSecretArn",
+        "value": "${var.rds_user_access_secret_arn}"
       }
     ],
-    "image": "cumuluss/async-operation:27",
+    "image": "jlkovarik/ecs-task-runner:41",
```

Comment:
Reiterating: can we publish to cumuluss/async-operation once integration tests are passing here?

Reply:
Hmm, I thought I answered that:

https://github.com/nasa/cumulus/pull/1982#discussion_r517687688